test/services/schema-file-updater.test.js

```diff
@@ -0,0 +1,207 @@
+const fs = require('fs');
+
+const logger = require('../../src/services/logger');
+const SchemaFileUpdater = require('../../src/services/schema-file-updater');
+const SchemaSerializer = require('../../src/serializers/schema');
+
+describe('services > schema-file-updater', () => {
+  const fsWriteFileSyncSpy = jest.spyOn(fs, 'writeFileSync').mockImplementation();
+  const loggerWarnSpy = jest.spyOn(logger, 'warn');
+
+  const meta = {
+    database_type: 'some-db-type',
+    liana: 'some-liana',
+    liana_version: 'some-liana-version',
+    engine: 'some-engine',
+    engine_version: 'some-engine-version',
+    framework: 'some-framework',
+    framework_version: 'some-framework-version',
+    orm_version: 'some-orm-version',
+  };
+  const schemaSerializer = new SchemaSerializer();
+  const { options: serializerOptions } = schemaSerializer;
+  const buildSchema = (collection, metas = meta) =>
+    new SchemaFileUpdater('test.json', collection, metas, serializerOptions).perform();
+
+  // NOTICE: Expecting fs.writeFileSync second parameter to be valid JSON
```

Comment:
Final dot is better but not that important IMO :)
```suggestion
  // NOTICE: Expecting `fs.writeFileSync` second parameter to be valid JSON.
```